src/plugins/discord/fetcher.js

```diff
@@ -159,7 +159,7 @@ export class Fetcher implements DiscordApi {
         id: x.user.id,
         username: x.user.username,
         discriminator: x.user.discriminator,
-        bot: x.user.bot || x.user.system || false,
+        bot: x.user.bot || false,
```

Comment:
This looks like a runtime change and probably shouldn’t be in a Flow
upgrade. Why is this here, and would it make sense to extract it?

Reply:
added the optional system member to the `User` type